modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -18,3 +18,9 @@ This feature is available only when you create applications using the *From Git*
 * Hover your cursor over the lower left icon on the Pod to see the name of the latest build and its status. The status of the application build is indicated as *New* (image:odc_build_new.png[title="New Build"]), *Pending* (image:odc_build_pending.png[title="Pending Build"]), *Running* (image:odc_build_running.png[title="Running Build"]), *Completed* (image:odc_build_completed.png[title="Completed Build"]), *Failed* (image:odc_build_failed.png[title="Failed Build"]), and *Canceled* (image:odc_build_canceled.png[title="Canceled Build"]).
 * Use the *Shortcuts* menu listed on the upper-right of the screen to navigate components in the *Topology* view.
 * Use the *List View* icon to see a list of all your applications and use the *Topology View* icon to switch back to the *Topology* view.
+* Use the *Display* dropdown menu to manipulate the various application groupings. These are automatically available depending on your project structure:
+** *Application Groupings*: Uncheck to disable application groupings and condense the application group into a card with an overview of an application group and alerts.
+** *Knative services*
+** *Helm releases*
+** *Operator groupings*
+* Use the search bar to highlight the component names that match the entry.
```

Comment:
Please check with the SME re the exact functionality of this search bar
Use the text in the field as the name of the search bar and make it bold since it is a GUI element.
s/highlight/see?
s/component names/components?

Reply:
It doesn't need to be based on the operators installed in the cluster,  they depend on the type of nodes shown in the graph. For example: If you have Knative service in the graph, then you will see Knative Services in the expand option. If you have not added any knative service to the namespace, then you will not see the option under the show/expand sections.

Reply:
Here in below screenshot i have added all the types of nodes in the graph, so the display options will show all the options 
![image](https://user-images.githubusercontent.com/9964343/87636064-bcb3b000-c75d-11ea-82b1-c2b00afa84fd.png)

Reply:
Thank you @karthikjeeyar this really helps!